helm/dex-app/values.yaml

```diff
@@ -51,6 +51,7 @@ oidc:
     dexK8SAuthenticator:
       clientAddress: ""
       clientSecret: "default-client-dex-authenticator-secret"
+  extraStaticClients: []
```

Comment:
Added custom static clients as an additional property of the values to separate them from the pre-defined ones and maintain backward compatibility at the same time.